tests/terraform/graph/variable_rendering/test_render_scenario.py

```diff
@@ -18,64 +18,83 @@
 
 class TestRendererScenarios(TestCase):
 
+    @mock.patch.dict(os.environ, {"CHECKOV_NEW_TF_PARSER": "False"})
```

Comment:
also in all of the places in this file, it's best to check it with the new parser because if and when we will remove the old one we will need to adjust them anyway